internal/criocli/criocli.go

```diff
@@ -725,6 +728,11 @@ func getCrioFlags(defConf *libconfig.Config) []cli.Flag {
 			Usage:   fmt.Sprintf("Determines whether we pin and remove IPC, network and UTS namespaces and manage their lifecycle (default: %v)", defConf.ManageNSLifecycle),
 			EnvVars: []string{"CONTAINER_MANAGE_NS_LIFECYCLE"},
 		},
+		&cli.BoolFlag{
+			Name:    "drop-infra-ctr",
+			Usage:   fmt.Sprintf("determines whether we drop the infra container when appropriate. Requires ManageNSLifecycle to be true (default: %v)", defConf.DropInfraCtr),
```

Comment:
instead of when appropriate should we reword to `when pod pid namespace is not shared`?

Reply:
I thought it a bit verbose for a cli flag TBH but yeah it would better match the config values